### PR TITLE
[NP-9967] no success message is thrown after resetting the password

### PR DESCRIPTION
**Pull request by Mykola97**
https://nanopay.atlassian.net/browse/NP-9967

### CODE REVIEW

File: src/foam/nanos/auth/resetPassword/ResetPassword.js [hunk 1]
```diff
@@ -93,8 +102,8 @@ foam.CLASS({
     {
       name: 'finalRedirectionCall',
       code: function() {
-        window.history.replaceState(null, null, window.location.origin);
-        location.reload();
+        this.ctrl.window.history.replaceState(null, null, this.ctrl.window.location.origin);
+        this.ctrl.stack.push(this.StackBlock.create({ view: { ...(this.ctrl.loginView ?? { class: 'foam.u2.view.LoginView' }), mode_: 'SignIn' }, parent: this.ctrl }));
```
Comment (sarthak-marwaha) [inline src/foam/nanos/auth/resetPassword/ResetPassword.js]: Are we trying to log the user out here?

Comment (KolombetM) [reply src/foam/nanos/auth/resetPassword/ResetPassword.js]: No, get back to log in screen but don't reload the app to keep success message

File: src/foam/nanos/auth/resetPassword/ResetPassword.js [hunk 2]
```diff
@@ -10,6 +10,15 @@ foam.CLASS({
 
   documentation: 'Reset Password Base Model',
 
+  imports: [
+    'ctrl',
```
Comment (kgrgreer) [inline src/foam/nanos/auth/resetPassword/ResetPassword.js]: Please don't import 'ctrl'. If you need something from ctrl then import that directly, like 'window' or 'stack'.

**Comment by noodlemoodle** [comment]
If this is an issue in the wizard flow: believe issue is due to resetpassword action not being executed.
executeactionssaver was updated recently which require changes to nanopay side - looking into a fix

**Comment by Mykola97** [comment]
> If this is an issue in the wizard flow: believe issue is due to resetpassword action not being executed. executeactionssaver was updated recently which require changes to nanopay side - looking into a fix

The issue is due to location.reload() function, which reload the app right after success message is shown, so user don't have time to see it

**Review by sarthak-marwaha — APPROVED** [review]

**Review by sarthak-marwaha — APPROVED** [review]

**Pull request merged**